auth0_flutter/lib/src/web/js_interop.dart

```diff
@@ -1,2 +1,86 @@
 // Here go the annotations –from the js package– that allow calling JS code
-// See https://pub.dev/packages/js#usage
\ No newline at end of file
+// See https://pub.dev/packages/js#usage
+
+// ignore_for_file: non_constant_identifier_names
+
+@JS('auth0')
+library auth0;
+
+import 'dart:js_util';
+
+import 'package:js/js.dart';
+
+@JS()
+@staticInterop
+@anonymous
+abstract class AuthorizationParams {
+  external factory AuthorizationParams(
+      {final String? audience, final String? redirect_uri});
+}
+
+extension on AuthorizationParams {
+  external String? audience;
+  external String? redirect_uri;
+
+  // https://github.com/dart-lang/sdk/issues/38445
+  // Workaround of setting property names doesn't seem to work. Tried:
+  // @JS('redirect_uri')
+  // external String? get redirectUri;
+
+  // @JS('redirect_uri')
+  // external set redirectUri(final String? redirectUri);
+}
```

Comment:
Most if not all the options will be supported here eventually, but having a couple to start off with helps to see the creation patterns. e.g. it took a while to see if we could support `redirectUri` in the Dart interop API but `redirect_uri` in JS-land. tl;dr is, we can't (yet).

Reply:
Does the workaround work when not using static interop?

Reply:
Unfortunately not. In fact it was this problem that prompted me to try static interop to see if it enabled it.